mathics/builtin/atomic/numbers.py

```diff
@@ -927,8 +914,8 @@ class Precision(Builtin):
       <dt>'Precision[$expr$]'
       <dd>examines the number of significant digits of $expr$.
     </dl>
-
-    <i>This is rather a proof-of-concept than a full implementation.</i>
+    <i>Notice that the result could be slighly different than the obtained
```

Comment:
backslash after "obtained" please.  For myself I go into Django and _look_ at the rendered output. it can help catching things like this.